DemoqaProject/tests/RadioButtonTest.cs

```diff
@@ -0,0 +1,25 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class RadioButtonTest : BaseSeleniumTest
+    {
+        [Test]
+        public void ClickAndCheck()
+        {
+            HomePage homePage = new HomePage();
+            homePage.NavigateToElementPage();
```

Comment:
Can you think about factory pattern to create and return a new object of a page when you navigate to it?

Reply:
In process